projects/charger/inc/generic_can_network.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+// Module to abstract the network CAN variant such that it shares an interface with UART CAN.
+// Requires network layer CAN to be running.
+
+#include "generic_can.h"
+
+#define NUM_GENERIC_CAN_NETWORK_RX_HANDLERS 5
```

Comment:
Should be using `NUM_GENERIC_CAN_RX_HANDLERS`?

Reply:
Done.